README.md

```diff
@@ -95,7 +95,96 @@ Following a certain file and folder structure keeps development, maintenance and
+
+  - #### .editorconfig
+
+    A definition of coding styles for different code editors and IDEs. You can specify what configuration (indent size, charset, trim whitespaces, ...) should be used for what files.
+
+    [Further info](http://editorconfig.org/)
+
+  - #### .eslintrc.js
+
+    [Eslint](https://eslint.org/) configuration. Primarily used to specify and configure rules of JavaScript linting.
+
+    [Further info](https://eslint.org/docs/user-guide/configuring)
+
+  - #### .gitattributes
+
+    Path-specific settings used by [Git](https://git-scm.com/). Settings that Git applies to certain subdirectories or subsets of files - for example EOL (End Of Line) setting.
+
+    [Further info](https://git-scm.com/docs/gitattributes)
+
+  - #### .gitignore
+
+    Contains patterns that are matched against file names in your [Git](https://git-scm.com/) repository to determine whether or not they should be ignored (commited).
+
+    [Further info](https://git-scm.com/docs/gitignore)
+
+  - #### .stylelintrc
+
+    [Stylelint](https://stylelint.io/) configuration. Primarily used to specify and configure rules of CSS (SCSS) linting.
+
+    [Further info](https://stylelint.io/user-guide/configuration/)
+
+  - #### circle.yml
+
+    [CircleCI](https://circleci.com/) configuration. How to set up and test your project.
+
+    [Further info](http://circleci.com/docs/1.0/configuration/)
+
+  - #### CONTRIBUTING.md
+
+    Project workflow guideline. How to create branches, write commits or assign pull requests.
+
+    [Further info](https://github.com/blog/1184-contributing-guidelines)
+
+  - #### gulpfile.js
+
+    [Gulp](https://gulpjs.com/) configuration and definition of automation tasks.
+
+    [Further info](https://www.sitepoint.com/introduction-gulp-js/)
+
+  - #### LICENSE
+
+    Open source license of [Git](https://git-scm.com/) repository. It enables others to freely use, change and distribute the project in the repository.
+
+    [Further info](https://help.github.com/articles/adding-a-license-to-a-repository/)
+
+  - #### package-lock.json
+
+    [NPM](https://www.npmjs.com/) manifest. Automatically generated with change of `node_modules` or `package.json`. Holds information about which versions of each dependency were installed in order to get consistent installs across machines.
+
+    [Further info](https://docs.npmjs.com/files/package-lock.json)
+
+  - #### package.json
+
+    [NPM](https://www.npmjs.com/) packages specifics. It lists the packages (with their versions) your project depends on.
+
+    [Further info](https://docs.npmjs.com/files/package.json)
+
+  - #### README.md
+
+    It's me!
+
+
+  - #### yarn.lock
+
+    [Yarn](https://yarnpkg.com/) manifest. Automatically generated with change of `node_modules` or `package.json`. Holds information about which versions of each dependency were installed in order to get consistent installs across machines.
```

Comment:
Maybe add that this file is generated only with `yarn` using `yarn add` if installing a new package or `yarn install` when installing packages from `package.json`